bin/pycbc_source_probability_offline

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+"""
+Compute source probabilities using mchirp estimation method for all events
+in a chunk with an IFAR above certain threshold.
+"""
+import os
+import h5py
+import json
+import tqdm
+import argparse
+import logging
+import numpy as np
+import pycbc
+from pycbc.io import hdf
+from pycbc.pnutils import mass1_mass2_to_mchirp_eta
+from pycbc import mchirp_area
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--trigger-file', required=True)
+parser.add_argument('--bank-file', required=True)
+parser.add_argument('--single-detector-triggers', nargs='+', required=True)
+parser.add_argument('--search-tag', required=True,
```

Comment:
maybe explain the help a bit more, eg 'String to add to the output file names, eg (...)'